scanner_capabilities.go

```diff
@@ -0,0 +1,141 @@
+package airscan
```

Comment:
Should this go into an internal package instead? Or should we unexport the types? I’m not sure we want to maintain this in our API.

Reply:
It could be a good idea to move them to their own package, so that others can use the XML types in their libraries, but not exporting them would be a lost opportunity to provide these AirScan / `eSCL` types.

Reply:
We can always export them at a later stage if people ask for it and have a good use-case. But we can’t easily go the other way around (unexporting them). Hence, I would prefer an internal package for now.

Reply:
Would an entirely new package in a different repo make more sense?

Something like `airscan_types` or similar

Reply:
Using a separate repository means using a separate Go module, and having a module just for the types seems like overkill to me. 

I really think using an internal package for the types is the best way forward for now.